dist/main.js

```diff
@@ -13,7 +13,8 @@ __webpack_require__.r(__webpack_exports__);
 /* harmony export */   "default": () => (__WEBPACK_DEFAULT_EXPORT__)
 /* harmony export */ });
 /* harmony import */ var _localStorage_js__WEBPACK_IMPORTED_MODULE_0__ = __webpack_require__(/*! ./localStorage.js */ "./src/modules/localStorage.js");
-/* harmony import */ var _assets_kebab_svg__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ../assets/kebab.svg */ "./src/assets/kebab.svg");
+/* harmony import */ var _assets_return_svg__WEBPACK_IMPORTED_MODULE_1__ = __webpack_require__(/*! ../assets/return.svg */ "./src/assets/return.svg");
+/* harmony import */ var _assets_kebab_svg__WEBPACK_IMPORTED_MODULE_2__ = __webpack_require__(/*! ../assets/kebab.svg */ "./src/assets/kebab.svg");
 function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
```

Comment:
- Just as the previous reviewers said, you should be able to click on the checkbox and the local storage should be updated with the correct value, either true or false, 
- Also at refresh of the webpage,  data should be read from the local storage so it maintains the proper value for the todo list as either completed or not (True or false in the local storage)
**Please try to implement the required changes**

**Tips (Algorithm )**

- Read to check if local storage exists
   - if local storage exists load all information from the local storage including the corresponding true or false
   - else create empty local storage, to be able to save the user data
- At onchange on the check box, the corresponding id of that checkbox should be gotten and the value of completed in the local storage should be changed to the opposite of what was previously there `!complete` use the not sign `!`

Feel free to ask any questions or clarification as needed. Happy coding

Reply:
Hello Ginohmk, Kindly note that these changes have been implemented🙏🙏.

when repo is cloned, kindly switch this branch which is called 'todofinal'....

this branch carries the changes.

Reply:
on click on the checkbox, values in the local storage changes to true or false either ways...

these changes also remains on refresh of the browser.

Reply:
I request that you please check my submission agan, thanks🙏🙏

@Ginohmk